_pages/ndtmetrics.md

```diff
@@ -159,3 +159,146 @@ WHERE
        AND web100_log_entry.snap.State <= 11))
   LIMIT 100
 ```
+
+## Packet Retransmission Rate
+
+NDT keeps track of the number of packets retransmitted during a test, in the web100 variable `web100_log_entry.snap.SegsRetrans`.
+
+Packet retransmission is computed as the ratio between the re-transmitted packets and all the transmitted packets.
```

Comment:
should be ....transmitted data packets.   Packets with zero length payload (e.g. pure ACKs) do not participate in the retransmission calculation